assets/src/dashboard/components/dropdown/index.js

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useCallback, useMemo, useRef, useState } from 'react';
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as DropDownArrow } from '../../icons/drop-down-arrow.svg';
+import { ReactComponent as DropUpArrow } from '../../icons/drop-up-arrow.svg';
+import useFocusOut from '../../utils/useFocusOut';
+import PopoverMenu from '../popover-menu';
+
+const DropdownContainer = styled.div`
+  position: relative;
+`;
+
+const Label = styled.label`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const InnerDropdown = styled.button`
+  align-items: center;
+  background-color: ${({ theme }) => theme.colors.gray25};
```

Comment:
The dashboard has this button with no background, let's add a `transparent` prop or export a `Transparent` version of the component.